client/payment-methods-map.tsx

```diff
@@ -29,17 +31,41 @@ import {
 
 const accountCountry = window.wcpaySettings?.accountStatus?.country || 'US';
 
-export interface PaymentMethodMapEntry {
-	id: string;
-	label: string;
-	description: string;
-	icon: ReactImgFuncComponent;
-	currencies: string[];
-	stripe_key: string;
-	allows_manual_capture: boolean;
-	allows_pay_later: boolean;
-	accepts_only_domestic_payment: boolean;
-}
+import type { PaymentMethodMapEntry } from './types/payment-methods';
+
+// Get any payment method definitions from the client.
+const PaymentMethodDefinitions =
+	typeof woopaymentsPaymentMethodDefinitions !== 'undefined'
+		? woopaymentsPaymentMethodDefinitions
+		: {};
+
+const convertedPaymentMethodDefinitions = Object.fromEntries<
+	PaymentMethodMapEntry
+>(
+	Object.entries( PaymentMethodDefinitions ).map( ( [ key, value ] ) => [
+		key,
+		{
+			id: value.id,
+			label: value.title,
+			description: value.description,
+			icon: ( { className } ) => (
+				<img
+					src={ value.icon }
+					alt={ value.title }
+					className={ classNames(
+						'payment-method__icon',
+						className
+					) }
+				/>
+			),
+			currencies: value.currencies,
+			stripe_key: value.stripeId,
```

Comment:
Sorry I didn't notice it in my earlier review - where is this `stripeId` property set in the `woopaymentsPaymentMethodDefinitions`? I see the other attributes are snake_case - can the same format be used?